pgml-dashboard/templates/layout/nav/top_web_app.html

```diff
@@ -0,0 +1,171 @@
+<% use crate::templates::components::GithubIcon; %>
+<% use crate::templates::components::PostgresLogo; %>
+<% 
+  let account_management_options = vec![
```

Comment:
These should be passed in through the context, not hardcoded.

Reply:
I changed this so it is passed in through context. I will need to come back through and move the rest of conditional links to context also.